### PR TITLE
added HelloWorld

### PR DESCRIPTION
**Pull request by NataliiaSH88**

### CODE REVIEW

**Review by AnnaChupina — APPROVED** [review]

**Review by emirayral — APPROVED** [review]
Looks great! You are hired!

**Review by Simona888 — COMMENTED** [review]
All good! :-)

**Review by Sabadi10 — COMMENTED** [review]
good job

**Review by Sabadi10 — APPROVED** [review]

File: src/Nataliia/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package Nataliia;
+
+public class HelloWorld {
```
Comment (camilorojas8) [inline src/Nataliia/HelloWorld.java:3]: looks good Nataliia, but please go back and add an '!' at the end to show pure excitement to the world

**Review by camilorojas8 — APPROVED** [review]

File: src/Nataliia/HelloWorld.java (hunk 1 above)
Comment (NataliiaSH88) [reply src/Nataliia/HelloWorld.java:3]: Camilo, probably I will do that in my next project..but not sure))

**Review by KateKing29 — APPROVED** [review]

**Review by pheonixzee — APPROVED** [review]

**Comment by NataliiaSH88** [comment]
finish

**Pull request closed without merging**